.buildkite/pipeline.yml

```diff
@@ -8,6 +8,33 @@ env:
   IMAGE_ID: xcode-15.4
 
 steps:
+  - label: Windows test inline Bash
+    command: |
+      echo "I'm about to exit 2. I expect this step to fail..."
+      exit 2
+    agents:
+      queue: windows
+
+  - label: Windows test Bash script
+    command: .buildkite/commands/command-that-fails.sh
+    agents:
+      queue: windows
```

Comment:
Succeeds in https://buildkite.com/automattic/ci-toolkit-buildkite-plugin/builds/1217

Might need additional config at call site to pass the failure along?